### PR TITLE
add cloud native GitHub and Azdo examples

### PR DESCRIPTION
**Pull request by lukebrogan-mend**

### CODE REVIEW

File: CI-CD/AzureDevOps/azure-pipelines_cloud_native_container_scan.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+trigger:
+- master
+
+pool:
+  vmImage: ubuntu-latest
+
+steps:
+- script: |
+    echo Downloading Mend CLI
+    sudo curl https://downloads.mend.io/production/unified/latest/linux_amd64/mend -o /usr/local/bin/mend && sudo chmod +x /usr/local/bin/mend
```
Comment (tabac-mend) [inline CI-CD/AzureDevOps/azure-pipelines_cloud_native_container_scan.yml:12]: sudo is not required as the command runs in a dockerized env

Comment (lukebrogan-mend) [reply CI-CD/AzureDevOps/azure-pipelines_cloud_native_container_scan.yml]: fixed, and added UUID parameter.

File: Mend CLI/azure-pipelines_cloud_native_container_scan.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+steps:
+- script: |
+    echo Downloading Mend CLI
+    curl https://downloads.mend.io/production/unified/latest/linux_amd64/mend -o /usr/local/bin/mend && chmod +x /usr/local/bin/mend
+    echo Start Mend CLI scan
+    mend image ubuntu:22.10
+  env:
+    MEND_EMAIL: $(EMAIL)
+    MEND_USER_KEY: $(USERKEY)
+    MEND_URL: https://saas-eu.whitesourcesoftware.com
```
Comment (tabac-mend) [inline Mend CLI/azure-pipelines_cloud_native_container_scan.yml:18]: Change to saas.whitesourcesoftware.com

File: Mend CLI/azure-pipelines_cloud_native_container_scan.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+    MEND_ORG_UUID: $(SCAORG)
```
Comment (tabac-mend) [inline Mend CLI/azure-pipelines_cloud_native_container_scan.yml:19]: We need to explain how to get the SCAORG UUID

File: Mend CLI/github-action-cloud-native-container-scan.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  push:
+    branches: [ master ]
+
+jobs:
+  mend-cli:
+    name: Mend-CLI
+    env:
+      MEND_EMAIL: ${{secrets.EMAIL}}
+      MEND_USER_KEY: ${{secrets.USERKEY}}
+      MEND_URL: https://saas-eu.whitesourcesoftware.com
```
Comment (tabac-mend) [inline Mend CLI/github-action-cloud-native-container-scan.yml:13]: saas.whitesourcesoftware.com

File: Mend CLI/github-action-cloud-native-container-scan.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+      MEND_ORG_UUID: ${{secrets.SCAORG}}
```
Comment (tabac-mend) [inline Mend CLI/github-action-cloud-native-container-scan.yml:14]: We need to explain how to get the SCAORG UUID

File: Mend CLI/gitlab-cloud_native_container_scan.yml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+
+stages:
+  - scan
+
+mend_scan:
+  stage: scan
+  variables:
+    MEND_EMAIL: $(EMAIL)
+    MEND_USER_KEY: $(USERKEY)
+    MEND_URL: https://saas-eu.whitesourcesoftware.com
```
Comment (tabac-mend) [inline Mend CLI/gitlab-cloud_native_container_scan.yml:12]: saas.whitesourcesoftware.com

**Review by tabac-mend — CHANGES_REQUESTED** [review]

**Pull request merged**